setup.py

```diff
@@ -86,28 +86,62 @@ def get_openmp_compile_args():
     extra_compile_args = ["-fopenmp"]
     if sys.platform == "darwin":
         extra_compile_args = ["-Xclang", "-fopenmp"]
+    elif sys.platform == "win32":
+        extra_compile_args = ["-openmp"]
     return extra_compile_args
 
 
 def get_openmp_link_args():
     extra_link_args = ["-fopenmp"]
     if sys.platform == "darwin":
         extra_link_args = ["-lomp"]
+    elif sys.platform == "win32":
+        extra_link_args = []
     return extra_link_args
 
 
+
+def collect_c_functions(source_files):
+    import re
+
+    cdecl_re = re.compile(r"(\S+)\s+CDECL\s+(\w+)")
```

Comment:
Some black magic going on here.

Reply:
Indeed. It's quite easy to collect all of the function names because CDECL occurs immediately prior to the function name.